src/main/minio.js

```diff
@@ -672,7 +672,11 @@ export default class Client {
 
     // Rename wrapper.
     var rename = () => {
-      fs.rename(partFile, filePath, cb)
+      if (partFile) {
+        fs.rename(partFile, filePath, cb)
+      } else {
+        cb(new Error('Object not downloaded'))
+      }
```

Comment:
@webfacebg thanks for the patch!
better way will be to handle `error`:
```
var rename = err => {
    if (err) return cb(err)
    fs.rename(partFile, filePath, cb)
}
```
can you resubmit the patch?